assets/scss/5-typography/_headline.scss

```diff
@@ -13,12 +13,9 @@
 .t-headline {
   @include font-setting('secondary');
   @include underlined-link-parent('true');
```

Comment:
I know we have helpers for these, but `t-headline` feels like it should contain some presets just for ease of use.